### PR TITLE
Created a model and schema folder

### PR DESCRIPTION
**Pull request by siddhant3030**

### CODE REVIEW

**Comment by aonomike** [comment]
Thanks @siddhant3030 for your hard work on this, please add some description to the PR and link it to an issue that is related to this task. Also, we can work on resolving the failing tests and ebert's concerns. Thanks!

**Comment by okothkongo** [comment]
Great work, am just wondering why we have directories named schema and model, technically I am of the opinion both context and schema forms the model. Assuming an artist would have an account, it is my opinion that it belongs to accounts context. Therefore we could have directory named accounts instead, for this am not sure, @aonomike or @FedericoEsparza is artist an account?.
Just do the quick fix on the test. Good job

**Comment by aonomike** [comment]
Hi @siddhant3030 we can pair on this one as well? would be great learn from it

File: lib/sing_for_needs/model/artist.ex [hunk 1]
```diff
@@ -0,0 +1,38 @@
+defmodule SingForNeeds.Model.Artist do
```
Comment (yakryder) [inline lib/sing_for_needs/model/artist.ex:1]: Why did you choose to remove the function documentation for this module?

**Review by yakryder — COMMENTED** [review]
Took a quick look.

@siddhant3030 Appreciate your wish to contribute! 

Although I am not certain if the PR template was available when you did this work, please complete it for future PRs.

Can you say something about what's motivating this PR? I'm in the process of learning Phoenix myself, but without more explanation I don't get it. I thought Phoenix 1.3 took us away from models and towards contexts

**Comment by siddhant3030** [comment]
> Hi @siddhant3030 we can pair on this one as well? would be great to learn from it

I thought we had discussed this. But sure I can explain to you again.

**Comment by yakryder** [comment]
This is moving against the grain relative to the larger ecosystem. There would need to be strong justification for it. I'm not seeing that.

@siddhant3030 We won't be using this. Thank you for your contributions. Please feel free to contribute again.

**Pull request closed without merging**